actions/websocket_actions.jsx

```diff
@@ -373,7 +373,50 @@ function handleChannelMemberUpdatedEvent(msg) {
     dispatch({type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBER, data: channelMember});
 }
 
-function handleNewPostEvent(msg) {
+export function debouncePostEvent(func, wait) {
+    let timeout;
+    let queue = [];
+    return function fx(...args) {
+        // Called when timeout triggered
+        const triggered = () => {
```

Comment:
A bit of a nit, but maybe we could move this out of the `fx` function and into `debouncePostEvent`? It doesn't use `args` directly, so it might clarify what happens when.

Reply:
Yup. It did use it at one point, will move out.